src/sentry/static/sentry/app/components/clippedBox.jsx

```diff
@@ -26,40 +28,64 @@ class ClippedBox extends React.Component {
     this.state = {
       clipped: this.props.defaultClipped,
       revealed: false, // True once user has clicked "Show More" button
+      renderedHeight: this.props.renderedHeight,
     };
   }
 
   componentDidMount() {
     const renderedHeight = ReactDOM.findDOMNode(this).offsetHeight; // eslint-disable-line react/no-find-dom-node
+    this.calcHeights(renderedHeight);
+  }
 
-    if (!this.state.clipped && renderedHeight > this.props.clipHeight) {
-      /*eslint react/no-did-mount-set-state:0*/
-      // okay if this causes re-render; cannot determine until
-      // rendered first anyways
-      this.setState({
-        clipped: true,
-      });
+  componentDidUpdate(_prevProps, prevState) {
+    if (prevState.renderedHeight !== this.props.renderedHeight) {
+      this.setRenderedHeight();
+    }
+
+    if (prevState.renderedHeight !== this.state.renderedHeight) {
+      this.calcHeights(this.state.renderedHeight);
     }
-  }
 
-  componentDidUpdate() {
     if (this.state.revealed || !this.state.clipped) {
       return;
     }
 
     const renderedHeight = ReactDOM.findDOMNode(this).offsetHeight; // eslint-disable-line react/no-find-dom-node
+
     if (renderedHeight < this.props.clipHeight) {
       this.reveal();
     }
   }
 
+  setRenderedHeight() {
+    this.setState({
+      renderedHeight: this.props.renderedHeight,
+    });
+  }
+
+  calcHeights(renderedHeight) {
+    if (!this.state.clipped && renderedHeight > this.props.clipHeight) {
+      /*eslint react/no-did-mount-set-state:0*/
+      // okay if this causes re-render; cannot determine until
+      // rendered first anyways
+      this.setState({
+        clipped: true,
+      });
+    }
+  }
+
   reveal = e => {
     e?.stopPropagation();
+    const {onRevel} = this.props;
```

Comment:
```suggestion
    const {onReveal} = this.props;
```

Reply:
It doesn't look like this is being used anywhere? Should these changes be in this PR?